### PR TITLE
Update accessing-traces.md

### LINKED ISSUE #4249: "Access Request Traces" doc links only work for Istio
**Describe the change you'd like to see**
[Access Request Traces](https://knative.dev/docs/serving/accessing-traces/) has links to Zipkin and Jaeger that only work if deployed to an Istio specific namespace eg "http://localhost:8001/api/v1/namespaces/istio-system/services/zipkin:9411/proxy/zipkin/", which assumes `istio-system` as the namespace.

Ideally, our instructions should be generic and work regardless of where Zipkin and Jaeger are installed

### PR DESCRIPTION
**Pull request by xtreme-sameer-vohra**
Fixes #4249 
Update instructions so they are general as opposed to Istio specific.

Happy to cherry-pick this for the 0.25 release if this gets merged before the next release is cut.

### CODE REVIEW

File: docs/serving/accessing-traces.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ In order to access request traces, you use the Zipkin visualization tool.
     reasons, the Zipkin UI is exposed only within the cluster.
 
 1.  Navigate to the
```
Comment (snneji) [inline docs/serving/accessing-traces.md:31]:
```suggestion
1. Access the Zipkin UI at the following URL:

    ```
    http://localhost:8001/api/v1/namespaces/<namespace>/services/zipkin:9411/proxy/zipkin/
    ```
    Where `<namespace>` is the namespace where Zipkin is deployed, for example, `knative-serving`.
```

File: docs/serving/accessing-traces.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ In order to access request traces, you use the Zipkin visualization tool.
...
-    [Zipkin UI](http://localhost:8001/api/v1/namespaces/istio-system/services/zipkin:9411/proxy/zipkin/).
-
+    Access the Zipkin UI at http://localhost:8001/api/v1/namespaces/REPLACEME/services/zipkin:9411/proxy/zipkin/
+    Replace `REPLACEME` in the link with the namespace where zipkin is deployed (eg. knative-serving).
```
Comment (snneji) [inline docs/serving/accessing-traces.md:33]:
```suggestion

```

File: docs/serving/accessing-traces.md [hunk 3]
```diff
@@ -50,9 +50,9 @@ In order to access request traces, you use the Jaeger visualization tool.
     This command starts a local proxy of Jaeger on port 8001. For security
     reasons, the Jaeger UI is exposed only within the cluster.
 
-1.  Navigate to the
-    [Jaeger UI](http://localhost:8001/api/v1/namespaces/istio-system/services/jaeger-query:16686/proxy/search/).
-
+1.  Access the Zipkin UI at http://localhost:8001/api/v1/namespaces/REPLACEME/services/jaeger-query:16686/proxy/search/
+    Replace `REPLACEME` in the link with the namespace where Jaeger is deployed (eg. knative-serving).
```
Comment (snneji) [inline docs/serving/accessing-traces.md:54]:
```suggestion
1.  Access the Jaeger UI at the following URL:

    ```
    http://localhost:8001/api/v1/namespaces/<namespace>/services/jaeger-query:16686/proxy/search/
    ```
    Where `<namespace>` is the namespace where Jaeger is deployed, for example, `knative-serving`.
```

**Review by snneji — CHANGES_REQUESTED** [review]
Thanks for the PR! I think it might be easier for users to copy the addresses for the Zipkin and Jaeger UIs if they were in code boxes. 

Also, there is a whitespace error.

**Comment by xtreme-sameer-vohra** [comment]
> Thanks for the PR! I think it might be easier for users to copy the addresses for the Zipkin and Jaeger UIs if they were in code boxes.
> 
> Also, there is a whitespace error.

Thanks for the suggestions :)

**Review by snneji — APPROVED** [review]

**Pull request merged**